tests/main/interfaces-block-devices-zfs/task.yaml

```diff
@@ -0,0 +1,50 @@
+summary: Ensure correctness of block-devices interface.
+
+details: |
+    The block-devices interface allows direct access to block devices, such as
+    /dev/sda, and optionally to partitions on said devices, e.g. /dev/sda1.
+
+systems:
+    # limit to systems where we know ZFS support is present OOTB
+    - ubuntu-2*
+
+prepare: |
+    dd if=/dev/zero of=disk bs=1M count=2048
```

Comment:
What's the minimum size of a block device that one can put ZFS on top of? Do we need all of 2GB here?